File: Latihan_B/cpp/FootballTeamList.cpp
#include <string>
#include <iostream>

using namespace std;

class FootballTeamList
{
    private:
        string team;
        string country;
        int year;
        string player;

    public:
        // constructor
        FootballTeamList(string team, string country, int year, string player){
            this->team = team;
            this->country = country;
            this->year = year;
            this->player = player;
        }

        FootballTeamList(){
        }

        // set Team Name
        void setTeam(string team)
        {
            this->team = team;
        }
        // set Team Name
        string getTeam()
        {
            return this->team;
        }

        // set Country
        void setCountry(string country)
        {
            this->country = country;
        }
        // set Country
        string getCountry()
        {
            return this->country;
        }

        // set Year
        void setYear(int year)
        {
            this->year = year;
        }
        // set Year
        int getYear()
        {
            return this->year;
        }

        // set Player Name
        void setPlayer(string player)
        {
            this->player = player;
        }
        // set Player Name
        string getPlayer()
        {
            return this->player;
        }

        // destructor
        ~FootballTeamList(){
        };
};

File: Latihan_B/java/FootballTeamList.java
public class FootballTeamList 
{
    private String team;
    private String country;
    private String year;
    private String player;

    // with constructor
    public FootballTeamList(String team, String country, String year, String player){
        this.team = team;
        this.country = country;
        this.year = year;
        this.player = player;
    }

    public FootballTeamList(){
    }

    // set Team Name 
    public void setTeam(String team) {
        this.team = team;
    }
     // get Team Name 
     public String getTeam() {
        return team;
    }

    // set Country 
    public void setCountry(String country) {
        this.country = country;
    }
    // get Country 
    public String getCountry() {
        return country;
    }

    // set Year 
    public void setYear(String year) {
        this.year = year;
    }
    // get Year 
    public String getYear() {
        return year;
    }
    
    // set Country 
    public void setPlayer(String player) {
        this.player = player;
    }
    // get Country 
    public String getPlayer() {
        return player;
    }
}